app/controllers/articles_controller.rb

```diff
@@ -124,6 +124,13 @@ def update
     end
   end
 
+  def toggle_mute
```

Comment:
I think ideally this has a RESTful resource controller like `ArticleMutes` which has `create` and `destroy` (or just an `update` action which sort of acts as a "toggle")

Since we have other exceptions, I'd be okay with leaving this for now. But it definitely needs to be changed away from a `GET` as described elsewhere.